rn/Teacher/src/common/components/DescriptionDefaultView.js

```diff
@@ -32,7 +33,7 @@ type Props = {
 
 export default class DescriptionDefaultView extends PureComponent<Props, any> {
   static defaultProps = {
-    text: 'Help your students with this assignment by adding instructions.',
+    text: i18n('Help your students with this assignment by adding instructions.'),
```

Comment:
still probably needs a translation, but at least it will get sent to the translators now.